### ISSUE TITLE
CTRL+PGUP and CTRL+PGDOWN not working for this Theme.

### ISSUE THREAD

**Issue by cyraid** [granella/Netbeans-Darcula-Theme#3]
`Using Netbeans Development (8.2) on Ubuntu 16.04 64-bit (amd64) with Oracle JDK 1.8.0_101-b13 (64-bit).`

CTRL+PGUP and CTRL+PGDOWN for Next/Previous Tab do not work with this Theme I found, but do work with the GTK+ Theme, for example.

**Comment by ghost** [comment]
I don't think this is the fault of Darcula.  I don't have Darcula, but I've just found this same problem trying out the Dark Nimbus look & feel in Netbeans Build 201609300101 on xubuntu 16.04.2 with xfce.  (The keys work fine in Dark Metal.)
